docs/resources/client-start-help

```diff
@@ -25,7 +25,8 @@ Flags:
       --tbtc.preParamsGenerationTimeout duration   tECDSA pre-parameters generation timeout. (default 2m0s)
       --tbtc.preParamsGenerationDelay duration     tECDSA pre-parameters generation delay. (default 10s)
       --tbtc.preParamsGenerationConcurrency int    tECDSA pre-parameters generation concurrency. (default 1)
-      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default number of cores)
+      --tbtc.keyGenerationConcurrency int          tECDSA key generation concurrency. (default 10)
```

Comment:
This line in the documentation is tricky as we want to have a custom default value label, but it's not supported by cobra. Would you mind reverting this line?

Reply:
Yeah, good catch. I did look at `--developer.bridgeAddress string ` added and overlooked this change. Fixed in 0d0fced97c205711b5ee8f396916b9f409579fbd.